ansible/services.yml

```
@@ -0,0 +1,152 @@
+---
+- hosts: all
+
+  vars:
+    IP: "0.0.0.0"
+    PORT: "8000"
+
+  tasks:
+    # Runserver service
+    - name: Deploy runserver
+      copy:
+        src: runserver.service
+        dest: /etc/systemd/system/
+        force: yes
+      become: yes
+
+    - name: Set ip:port configuration in service file
+      replace:
+        path: /etc/systemd/system/runserver.service
+        regexp: "ip:port"
+        replace: "{{IP}}:{{PORT}}"
+      become: yes
+
+    - name: Systemd daemon reload
+      systemd:
+        daemon_reload: yes
+      become: yes
+
+    - name: Disable runserver service
+      service:
+        name: runserver
+        enabled: no
+      become: yes
+
+    - name: Enable runserver service
+      service:
+        name: runserver
+        enabled: yes
+      become: yes
+
+    - name: Start runserver
+      systemd:
+        name: runserver
+        state: started
+      become: yes
```

Czy nie można zwięźlej?
```suggestion
    - name: Enable&start Django Runserver service
      service:
        name: runserver
        enabled: yes
        state: started
      become: yes
```